addons/okr-daily-script.kp/knowledge.md

```diff
@@ -123,9 +123,9 @@ These are the aggregations / joins that we **don't** want to do in re:dash.
    * 1  -> has a custom theme
    * -1 -> has default theme
    * 0  -> changed from default to custom on this date
-* To facilitate total population percentages, each submission date has two static fields
-  + n_custom_theme_clients (# distinct clients on that day with a custom theme)
-  + n_clients (# distinct total clients on that date)
+* To facilitate total population percentages, each submission date/channel has two static fields
```

Comment:
Please also update the comment just after the `default_theme_id` line below to reflect the addition of channel info.